src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -33,6 +33,7 @@ object JesBackend {
   lazy val GoogleApplicationName = JesConf.getString("applicationName")
   lazy val EndpointUrl = new URL(JesConf.getString("endpointUrl"))
   lazy val JesConnection = JesInterface(GoogleApplicationName, EndpointUrl)
+  lazy val DockerHubCredentials = Try(JesConf.getString("dockerhubCredentialsPath")).toOption
```

Comment:
Just for discussion here... looking at the API, config.getString() can only reasonably throw a ConfigException.Missing.  It could of course throw a Throwable, but Try won't capture those.  If I came across this code in the future I would wonder... huh, what am I missing here... so I think your original suggestion is actually just as robust but more clear and concise about what it's doing.  Thoughts?